appliance/postgresql/runner.go

```diff
@@ -83,14 +91,14 @@ func startLeader() (*exec.Cmd, <-chan struct{}) {
 }
 
 func register(attrs map[string]string) {
-	err := discoverd.RegisterWithAttributes(*serviceName, addr, attrs)
+	err := heartbeater.SetMeta(attrs)
 	if err != nil {
 		log.Fatalln("discoverd registration error:", err)
```

Comment:
This code will be thrown out in the next week, I didn't change anything extra to avoid increasing the diff size.